plugins/data/dataframes/src/hopeit/dataframes/datablocks.py

```diff
@@ -64,7 +67,13 @@ def to_dataobjects(
 
 class DataBlocks(Generic[DataBlockType, DataFrameType]):
```

Comment:
Needs docstring

Reply:
Done in https://github.com/hopeit-git/hopeit.engine/pull/205/commits/4e580c249f30d650ecc463ef32f0541e6575b949